src/sentry/consumers/__init__.py

```diff
@@ -405,6 +405,34 @@ def get_stream_processor(
             f"Invalid consumer definition configured for {consumer_name}"
         ) from e
 
+    post_process_forwarders = [
```

Comment:
those should never have dlq? are they some kind of exception?

Reply:
I think it's not strictly necessary on post process forwarders since they consume the exact same topic as the Snuba consumers, and those bad messages on the topic should already get DLQed by the main consumer. They only exist for synchronization purposes to ensure post processing doesn't run before the clickhouse write.

Reply:
Doesn't the post process forwarder skip parsing the payload entirely anyway ? 
I think it only uses headers and they are not part of the schema.